### ISSUE TITLE
Pretrained model contains custom loss, during loading a saved model from h5 file TRAINING process is starting

### ISSUE THREAD

**Issue by akbacak** [keras-team/keras#10920]
Hi Keras Team!

i have a  custom loss function a my pretrained model. When i load trained model from h5 file TRAINING process starts from scratch

`from custom_loss48 import c_loss`
`cnn_model = load_model('custom_loss48.h5', custom_objects={'c_loss': c_loss} )`

Is this a bag of Keras ?

**Comment by akbacak** [comment]
loading from json model is a solution
